controllers/os-common/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
```

Comment:
See above, don't do that, please.

Reply:
Can you please be more specific what change you suggest?  It is having common templates?

Reply:
>  In the future we plan to move out the extensions into dedicated repositories, so already having most of the files needed in place will make that easier.

--> Everything that we share (and that's already too much) will complicate this process. Let's try to not share as much as possible so that the move-out gets easier.

Reply:
I understand sharing templates is problematic when splitting into multiple repositories. I will move the templates.

Reply:
With respect of the os-common. The idea is to have a common implementation that can be used for other controllers, including the existing alicloud controller. This can be considered a library, much in the same way that the share code in `gardener-extensions/pkg/controller'.  Having a separated repository for each controller will not be more complicated by the fact of using such common code.

Reply:
If you want to share code between OS controllers then https://github.com/gardener/gardener-extensions/tree/master/pkg/controller/operatingsystemconfig is the correct place (probably a subpackage there). Everything in the `controllers` folder is to be moved out in a dedicated repo, thus it should only contain real controllers and not libraries.